src/EnergyPlus/SwimmingPool.hh

```diff
@@ -162,6 +161,12 @@ namespace SwimmingPool {
         }
 
         void simulate(EnergyPlusData &EP_UNUSED(state), const PlantLocation &calledFromLocation, bool FirstHVACIteration, Real64 &CurLoad, bool RunFlag) override;
+        
+        void ErrorCheckSetupPoolSurface(std::string const Alpha1,
+                                        std::string const Alpha2,
+                                        std::string const cAlphaField2,
+                                        bool &ErrorsFound
+        );
```

Comment:
Another minor nitpick for later.  Try to start naming methods with `camelCase` instead of `PascalCase`.  Class names and namespaces should be `PascalCase`, but all variables and functions should be `camelCase`.